src/pulselistener/pulselistener/mercurial.py

```diff
@@ -157,13 +157,14 @@ def clean(self):
             await asyncio.sleep(1)
 
         # Build and commit try_task_config.json
+        build_target_phid = diff.get('build_target_phid')
         config_path = os.path.join(self.repo_dir, 'try_task_config.json')
         config = {
             'version': 2,
             'parameters': {
                 'target_tasks_method': 'codereview',
                 'optimize_target_tasks': True,
-                'phabricator_diff': diff['phid'],
+                'phabricator_diff': build_target_phid or diff['phid'],
```

Comment:
In https://github.com/mozilla/release-services/pull/2048/commits/9a6a1c09a3319b74b1c5a71a6eb2d8139b6afffd, you are assuming this is always a build target PHID.
So maybe you can rename it `phabricator_build_target`? And remove the `or diff['phid']`?

Reply:
That would involve a patch on Mozilla Central, as the list of parameters is strictly defined...

Reply:
OK, let's keep it like this then.